pages/service-usages/advanced-usages.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: page
+title: Trippin Advanced Usages
+permalink: odata-services/service-usages/trippin-advanced-usages/
+---
+
+## Trippin Advanced Usages
+
+### Singleton
+
+#### Requesting singleton
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Me
+```
+
+### Singleton
+
+#### Requesting property of Singleton
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/Me/AddressInfo
+```
+
+### Derived Entity Type
+
+#### Requesting Derived Entity Type
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('kristakemp')/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee
+```
+
+#### Requesting a Derived Entity Collection
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee
+```
+
+#### Filter on Derived Entity
+
+```
+GET http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People('kristakemp')/Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee?$filter=UserName eq 'mirsking'
+```
+
+#### Create a Derived Entity
+
+```javascript
+POST http://services.odata.org/TripPinRESTierService/(S(3mslpb2bc0k5ufk24olpghzx))/People
+{
+    "@odata.type": "Microsoft.OData.Service.Sample.TrippinInMemory.Models.Employee",
+    "UserName": "mirsking",
+    "FirstName": "Mirs",
+    "LastName": "King",
+    "AddressInfo": [
+    {
+      "Address": "Microsoft",
+      "City": {
+        "Name": "Shanghai",
+        "CountryRegion": "China"
+      }
+    }
+    ]
```

Comment:
Do we have derived type properties here? Have some property easier to know belongs to Employee like EmployeeSeat or something else.